x/evm/keeper/statedb.go

```diff
@@ -206,6 +243,9 @@ func (k *Keeper) DeleteAccount(ctx sdk.Context, addr common.Address) error {
 		return true
 	})
 
+	// clear code hash
+	k.SetCodeHash(ctx, addr, common.BytesToHash(types.EmptyCodeHash))
```

Comment:
This is bad practice. Please add a `DeleteCodeHash(ctx, addr)` to make it clear that that's the action being taken

Reply:
It's the same practice taken in the rest of the module - should I add corresponding methods for the `SetCode` and `SetState` methods too?

Reply:
yup, let's do that